tests/integration/targets/vmware_ops_provision_vm_test/tasks/post_update_settings_validation.yml

```diff
@@ -57,11 +40,20 @@
     label: Hard disk 1
   register: disk_info
 
+- name: Get VM Guest Name
+  community.vmware.vmware_vm_info:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    validate_certs: "{{ provision_vm_validate_certs }}"
+    vm_name: "{{ vm_update_name }}"
+  register: guest_name_info
+
 - name: "Validate updated configuration of '{{ vm_update_name }}' VM"
   ansible.builtin.assert:
     that:
-      - vm_info.virtual_machines[0].guest_fullname == guest_fullnames[vm_updated_guest_id]
-      - cpu_info.value.count == vm_updated_hardware.num_cpus
-      - cpu_info.value.cores_per_socket == vm_updated_hardware.num_cpu_cores_per_socket
-      - memory_info.value.size_MiB == vm_updated_hardware.memory_mb
+      - guest_name_info.virtual_machines[0].guest_fullname == guest_fullnames[vm_updated_guest_id]
```

Comment:
@mikemorency Should I open a bug that `vmware.vmware.guest_info` doesn't gather `guest_fullname`?

Reply:
Ill make one, i need to add some extra details to it anyway